src/core/utils/format-string.ts

```diff
@@ -21,3 +21,10 @@ export const valuePrimaryCtaField = (validators: IValidator[]): string => {
 export const formatValidatorName = (str: string, length: number): string => {
     return str.length > length ? str.slice(0, length) + '...' : str;
 };
+
+export const formatPlural = (value: number): string => {
+    if (value === 1) return '1st';
```

Comment:
Oh no :) will this work when we add other languages ? :)
The best approach here is to use our translation system, that way you will cover all cases. Maybe in this case is not applicable, but for instance you function will not handle correct 21, 22, 101, 102, 103... and many others. Our translation system is very capable. So you just have to add this in our en.ts file:

```ts
"ParentKey": {
   "ledgerSignButton": {
      ordinal: true,
      text: "Sign {{txNumber}}th transaction",
      forms: {
         one: "Sign {{txNumber}}st transaction",
         two: "Sign {{txNumber}}nd transaction",
         few: "Sign {{txNumber}}rd transaction",
         other: "Sign {{txNumber}}th transaction", // this is not needed since we have the default text above
    }
}
```
Or you can have only the numbers in a key and have the button text as "SIGN {{number}} transaction".

This way the ordinal numbers formatted correctly, like in this table https://linguapress.com/grammar/ordinals.htm and fully customisable for any language.